### PR TITLE
restore vehicle+wheel importers

### LINKED ISSUE #61733: Vehicle and wheel import hints do not work in v4
### Godot version

alpha 9

### System information

Fedora Silverblue 36, Ryzen 2400G

### Issue description

Importing vehicles and wheels using `-vehicle`/`-wheel` suffixes in Blender files does not work anymore in v4. The feature is still documented in the latest docs: https://docs.godotengine.org/en/latest/tutorials/assets_pipeline/importing_scenes.html#create-a-vehiclebody-vehicle

Not sure if it has been removed intentionally, if yes the docs need to be updated. But I'd really like to see this feature returned, especially since there is another bug with inherited scenes not being updated.

Godot 3 vs Godot 4:

Screenshot from 2022-06-05 10-44-06

### Steps to reproduce

1. Create meshes in blender with -vehicle and -wheel suffixes
2. Import the file into Godot

### Minimal reproduction project

_No response_

### PR DESCRIPTION
**Pull request by jgillich**
fixes #61733

Tested and it works as expected

### CODE REVIEW

**Comment by fire** [comment]
Are you ok checking if the tests still work? https://github.com/godotengine/godot-tests/tree/master/tests/blend_export

Wanted to show people the tests we made a long time ago.

**Review by fire — COMMENTED** [review]
Is the node modified by replaced by leaked?

**Comment by jgillich** [comment]
Which node? I don't see it

File: editor/import/resource_importer_scene.cpp [hunk 1]
```diff
@@ -658,6 +658,44 @@ Node *ResourceImporterScene::_pre_fix_node(Node *p_node, Node *p_root, HashMap<R
...
+		if (isroot) {
+			return p_node;
+		}
+
+		Node *owner = p_node->get_owner();
+		Node3D *s = Object::cast_to<Node3D>(p_node);
+		VehicleBody3D *bv = memnew(VehicleBody3D);
+		String n = _fixstr(p_node->get_name(), "vehicle");
+		bv->set_name(n);
+		p_node->replace_by(bv);
```
Comment (fire) [inline editor/import/resource_importer_scene.cpp:671]: Code above this memdeletes and wanted to check. This happens twice.

Comment (jgillich) [reply editor/import/resource_importer_scene.cpp:671]: `p_node` is appended to `bv` below; I don't think there is a leak here.

**Review by fire — APPROVED** [review]
I haven't had time to launch godot and test the test projects, but the paper review looks reasonable.

Please add a few visuals of that, but I think it's good.

**Comment by akien-mga** [comment]
Thanks! And congrats for your first merged Godot contribution :tada:

**Pull request merged**